shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/morePage/feedbackFormUrl.kt

```diff
@@ -1,12 +1,14 @@
 package com.mbta.tid.mbta_app.model.morePage
 
-fun feedbackFormUrl(translation: String) =
-    when (translation) {
-        "es" -> "https://mbta.com/appfeedback?lang=es-US"
-        "ht" -> "https://mbta.com/appfeedback-ht"
-        "pt-BR" -> "https://mbta.com/appfeedback?lang=pt-BR"
-        "vi" -> "https://mbta.com/appfeedback?lang=vi"
-        "zh-Hans-CN" -> "https://mbta.com/appfeedback?lang=zh-Hans"
-        "zh-Hant-TW" -> "https://mbta.com/appfeedback?lang=zh-Hant"
-        else -> "https://mbta.com/appfeedback"
+fun feedbackFormUrl(translation: String): String {
+    val formLink = "https://mbta.com/androidappfeedback"
```

Comment:
We do not want to send iOS users to the Android app feedback form.

Reply:
😱 Should have checked that this is in shared, thank you!